oommfc/drivers/driver.py

```diff
@@ -66,6 +66,7 @@ def drive_kwargs_setup(self, drive_kwargs):
         drive_kwargs.setdefault("output_step", False)
         drive_kwargs.setdefault("n_threads", None)
         drive_kwargs.setdefault("compute", None)
+        return drive_kwargs
```

Comment:
This is neccessary for oommfc/scripts/driver.py to see a change in kwargs. We would also need to update micromagneticmodel/driver.py (Line 129) so that kwargs is updated in the whole call stack.

Reply:
This line is not required: the method takes a single dict, not individual keyword arguments (i.e. `drive_kwargs` and **not** `**drive_kwargs`). Therefore, we operate on the same dictionary that the calling method (`micromagneticmodel.Driver::drive`) is referencing by the name `kwargs` and any changes to this dictionary inside `drive_kwargs_setup` are "automatically" visible to the calling method because we operate on the same object.

Note, this is inconsistent with how `_checkargs` is used and should be unified in a (separate internal) refactoring/cleanup.

Reply:
I have modified the code so that `_checkargs` modifies the dict `kwargs` that is passed in instead of creating a new object. This is in line with the assumptions made in `micromagneticmodel`.